builder.config.json

```diff
@@ -277,29 +277,54 @@
       {
         "id": "10",
         "name": "name",
-        "display_label": "Name (Text)"
+        "display_label": "Name (Text)",
+        "data_type": "text"
       },
       {
         "id": "11",
         "name": "comment",
-        "display_label": "Comment (Text)"
+        "display_label": "Comment (Text)",
+        "data_type": "text"
       },
       {
         "id": "12",
-        "name": "start_date",
-        "display_label": "Start Date (Date)"
+        "name": "age",
+        "display_label": "Age (Number)",
+        "data_type": "number"
+      },
+      {
+        "id": "13",
+        "name": "phone_number",
+        "display_label": "Phone number (phone)",
+        "data_type": "phone"
+      },
+      {
+        "id": "14",
+        "name": "favourite_fruits",
+        "display_label": "Favourite fruits (Multiple_choice)",
+        "data_type": "multiple_choice"
+      },
+      {
+        "id": "15",
+        "name": "address",
+        "display_label": "Address (Location)",
+        "data_type": "location"
+      },
+      {
+        "id": "16",
+        "name": "avatar",
+        "display_label": "Avatar (Media)",
+        "data_type": "media"
       }
     ],
-    "subscriberPropertyFieldDataTypes": [
-      "text",
-      "location",
-      "number",
-      "map_coordinates",
-      "boolean",
```

Comment:
As I understand, we don't have properties of these types. Should we keep them here as `map_coordinates: []`, or remove from `src/Subscribers/SubscriberPropertyField.php`?

Reply:
@object-required , in this repo we're specifying the community version, the list in `builder.config.json` is just a example of what would be a `subscriberPropertyFields`. I think, providing different list here and in `SubscriberPropertyField.php` is helping devs to understand those lists are totally different. If they are the same, it doesn't make sense to override the list in voto5.